lib/task.js

```diff
@@ -99,6 +99,23 @@ async function buildVolumeBindings(taskVolumeBindings, volumeCache, taskScopes)
   return [caches, bindings];
 }
 
+function buildDeviceBindings(devices) {
+  let deviceBindings = [];
+  for (let deviceType in devices) {
+    let device = devices[deviceType];
+    device.mountPoints.forEach((mountPoint) => {
+      deviceBindings.push(
+        {
+          PathInContainer: mountPoint,
```

Comment:
ahh- we could attempt to mount to default 1-2/etc.. locations but it appears that our test harnesses are smart enough to pick any number